src/sinks/sematext.rs

```diff
@@ -0,0 +1,112 @@
+use crate::{
+    sinks::elasticsearch::ElasticSearchConfig,
+    sinks::util::Compression,
+    topology::config::{DataType, SinkConfig, SinkContext},
+};
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Deserialize, Serialize)]
+pub struct SematextConfig {
+    cloud: Cloud,
+    #[serde(flatten)]
+    inner: ElasticSearchConfig,
```

Comment:
I think instead of trying to _not_ let the user configure it, just tell them it doesn't exist in the docs? I'm not sure how to ignore certain things without having to re implement the config struct each time.

I could go either way though, I think if we just don't tell people its possible then they won't try it? similar to `doc(hidden)`. 🤷‍♂

Reply:
Could we inspect the config like you're doing below to see if they've configured a region or anything like that and provide a suitable error message? Not ideal but could be fine.